PRISM_maproom/diversity_maproom.py

```diff
@@ -172,17 +171,14 @@ def colorMap(date, myspecies):
     [Input("date_dropdown", "value")],
 )
 def outagePoints(date):
-    dfLoc = dfJoined.loc[dfJoined["date"] == date]
-    dfLoc = dfLoc[["geometry", "eBird.DP.RF"]]
-    dfLoc = dfLoc.rename(columns={"eBird.DP.RF": "diversity"})
     # outage data
     dateSince = datetime.strptime("2010-01-01", "%Y-%m-%d")
     dateSelect = datetime.strptime(date, "%Y-%m-%d")
-    dateDiff = (dateSince - dateSelect).days  # beginning of the day we are looking at
+    dateDiff = (dateSelect - dateSince).days  # beginning of the day we are looking at
```

Comment:
How was it not obviously broken before if dateDiff was negative?

Reply:
dateDiff can be negative. It's fine if the date is prior to 2010. So no reason to break but actually since the outage data starts in 2013, dateDiff for 2005 was positive and actually selecting outage data in 2015 I guess